src/main/java/edu/harvard/h2ms/web/controller/ReportController.java

```diff
@@ -0,0 +1,33 @@
+package edu.harvard.h2ms.web.controller;
+
+import java.io.IOException;
+
+import javax.servlet.http.HttpServletResponse;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+import edu.harvard.h2ms.service.ReportService;
+
+@RestController
+@RequestMapping(path="/api/reports")
+public class ReportController {
+	
+	@Autowired
+	private ReportService reportService;
+	
+	
+	@RequestMapping(value = "/events", method = RequestMethod.GET)
+	public void getEventsReport(HttpServletResponse response){
+		response.setContentType("text/plain; charset=utf-8");

```

Comment:
CSV should be `text/csv`

Reply:
httpHeaders.add("Content-Type", "text/csv; charset=utf-8");  maybe application/csv?